cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Actions.hs

```diff
@@ -46,52 +47,52 @@ data GovernanceActionCmds era
 
 data GoveranceActionUpdateCommitteeCmdArgs era
   = GoveranceActionUpdateCommitteeCmdArgs
-    { ucNetwork :: Ledger.Network
-    , ucDeposit :: Lovelace
-    , ucReturnAddress :: AnyStakeIdentifier
-    , ucProposalUrl :: ProposalUrl
-    , ucProposalHashSource :: ProposalHashSource
-    , ucOldCommittee :: [VerificationKeyOrHashOrFile CommitteeColdKey]
-    , ucNewCommittee :: [(VerificationKeyOrHashOrFile CommitteeColdKey, EpochNo)]
-    , ucRequiredQuorum :: Rational
-    , ucPreviousGovActionId :: Maybe (TxId, Word32)
-    , ucFilePath :: File () Out
-    } deriving Show
+      { networkId               :: !Ledger.Network
```

Comment:
So I take that having multiple fields with the same name in a module is fine now?

Reply:
I think this is okay for command argument types because the qualifications and record field matching allows us to avoid naming conflicts.